src/scales/scale.category.js

```diff
@@ -48,71 +49,54 @@ module.exports = Scale.extend({
 		return me.ticks[index - me.minIndex];
 	},
 
-	// Used to get data value locations.  Value can either be an index or a numerical value
-	getPixelForValue: function(value, index) {
+	_getParams: function() {
 		var me = this;
 		var offset = me.options.offset;
 		// 1 is added because we need the length but we have the indexes
-		var offsetAmt = Math.max((me.maxIndex + 1 - me.minIndex - (offset ? 0 : 1)), 1);
+		var range = Math.max((me.maxIndex + 1 - me.minIndex - (offset ? 0 : 1)), 1);
```

Comment:
I'd probably remove the parens around `me.maxIndex + 1 - me.minIndex - (offset ? 0 : 1)`